composer.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "krakjoe/pcov",
```

Comment:
Please verify vendor namespace is acceptable; this would need @krakjoe to submit this to Packagist; can change it to something else like `pcov/pcov` if preferred

Reply:
IMHO  `pecl/pcov `will be better.